src/components/ProfileInformation/ProfileInformation.spec.tsx

```diff
@@ -38,6 +44,37 @@ describe('ProfileInformation', () => {
         expect(getByText(avatarName)).toBeInTheDocument()
         expect(getByTestId(anAddress)).toBeInTheDocument()
       })
+
+      it('should render the actions buttons', async () => {
```

Comment:
Would you mind creating a new test where the user is not logged in? We should be showing the share and copy link all the time, even when the user is not logged in.